R/utils.r

```diff
@@ -190,5 +190,8 @@ modify_list <- function(x, y) {
   if (is.null(y)) {
     return(x)
   }
+  if (is.null(x)) {
```

Comment:
Could you drop this change by returning `list()` instead of `NULL` in `find_template_config()`?